lib/main_common.pm

```diff
@@ -2212,9 +2212,10 @@ sub load_security_tests_mok_enroll {
 
 sub load_security_tests_ima_measurement {
     loadtest "security/ima/ima_setup";
-    loadtest "security/ima/ima_measurement";
-    loadtest "security/ima/ima_kernel_cmdline_template";
-    loadtest "security/ima/ima_kernel_cmdline_hash";
+    #loadtest "security/ima/ima_measurement";
+    #loadtest "security/ima/ima_kernel_cmdline_template";
+    #loadtest "security/ima/ima_kernel_cmdline_hash";
+    loadtest "security/ima/ima_measurement_audit";
```

Comment:
the three "#"(s) should be deleted :)

Reply:
My mistake. It is for debugging. Modified.

Reply:
Thanks for the update:)